File: miniproject/src/main/java/com/targetindia/miniproject/model/Customer.java
package com.targetindia.miniproject.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Customer {
    private int id;
    private String name;
    private String city;
    private String email;
    private String phone;
}

File: day-12/workspace/kotlin-oop/src/main/kotlin/com/targetindia/model/Customer.kt
package com.targetindia.model

class Customer {
    var name = ""
    var city = ""
    var email = ""

    constructor()

    constructor(name: String, city: String = "Bangalore", email: String) {
        this.name = name
        this.city = city
        this.email = email
    }

    fun getAsCsv(): String {
        return "$name,$city,$email"
    }

    fun getAsJson(): String {
        return """{
            |"name": "$name",
            |"city": "$city",
            |"email": "$email"
            |}""".trimMargin()
    }

    fun print(): Unit {
        println("Name    : $name")
        println("City    : $city")
        println("Email   : $email")
    }
}